nondex-maven-plugin/src/it/excluded-groups-it/pom.xml

```diff
@@ -18,24 +18,6 @@
       <module>module2</module>
     </modules>
 
-    <build>
```

Comment:
@azy2 why did you remove this? Because you remove this, the build is broken. If you put it back, you will get some other issue, because module 2 has failing tests with nondex. Please debug that more.